oex_plugin/openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -124,4 +125,18 @@ def _update_additional_extended_profile_fields_if_needed(data, user):
     """
     if 'english_proficiency' in data:
         user.extended_profile.english_proficiency = data['english_proficiency']
-        user.extended_profile.save()
+    if 'organization' in data:
+        if org_data := data['organization']:
+            org_label = org_data.get('org_label')
```

Comment:
is there a way to get None here?

Reply:
@DmytroAlipov please react

Reply:
We cannot get rid of this block, since we need to understand exactly whether a request came with such a field, or whether this field is not in the request at all. The logic of the user’s connection with the organization depends on this.

Reply:
I think you thought about a different question when commenting on this one.

I wonder if there is any change where we will get here and `org_data.get('org_label')` will return `None`?
If so, what will happen?

Reply:
oh yeah, I didn't understand correctly. If these organization data come, then `org_label` will be present. `None` is impossible to get here